packages/message-list/src/vaadin-message-list.js

```diff
@@ -85,28 +91,29 @@ class MessageList extends KeyboardDirectionMixin(ElementMixin(ThemableMixin(Poly
           display: none !important;
         }
       </style>
-      <div part="list" role="list">
```

Comment:
Why do we need to expose this div? I don't see obvious problems with accessing it with the `::part` selector. Moving it to the Light DOM leads to having to use not dedicated selectors to set default styles which is not very good:

```css
registerStyles(
  'vaadin-message-list',
  css`
    ::slotted([role='list']) {
      padding: 8px 0;
    }
```

Reply:
It's possible that element with `role="list"` in Shadow DOM might not work with slotted items. 
But this is just my assumption, will test how this works with actual screen readers.

Reply:
Confirmed that screen readers work with messages in light DOM and list in shadow DOM:

### NVDA

![list-nvda](https://user-images.githubusercontent.com/10589913/195146540-fbc6bf6c-86b9-4b70-a437-217b7e1bd936.png)

### JAWS

![list-jaws](https://user-images.githubusercontent.com/10589913/195146535-9162bd25-7efb-4585-83b9-e9408135659a.png)

Updated the PR accordingly, to revert changes related to `role="list"` element.